tests/models/BaseDocTest.php

```diff
@@ -17,15 +17,22 @@ class BaseDocTest extends TestCase
      */
     public function testExtractFirstSentenceWithBackticks()
     {
-        $initialText = 'fallback host info (e.g. `http://www.yiiframework.com`) used when ' .
-            '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] is invalid. This value will replace ' .
-            '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] before [[$denyCallback]] is called to make sure that ' .
-            'an invalid host will not be used for further processing. You can set it to `null` to leave ' .
-            '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] untouched. Default value is empty string (this will ' .
-            'result creating relative URLs instead of absolute).';
+        $initialText = 'fallback host info (e.g. `http://www.yiiframework.com`) used when '
+            . '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] is invalid. This value will replace '
+            . '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] before [[$denyCallback]] is called to make sure that '
+            . 'an invalid host will not be used for further processing. You can set it to `null` to leave '
+            . '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] untouched. Default value is empty string (this will '
+            . 'result creating relative URLs instead of absolute).';
         $actualFirstSentence = BaseDoc::extractFirstSentence($initialText);
-        $expectedFirstSentence = 'fallback host info (e.g. `http://www.yiiframework.com`) used when ' .
-            '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] is invalid.';
+        $expectedFirstSentence = 'fallback host info (e.g. `http://www.yiiframework.com`) used when '
+            . '[[\yii\web\Request::$hostInfo|Request::$hostInfo]] is invalid.';
+        $this->assertEquals($expectedFirstSentence, $actualFirstSentence);
+        
+        $initialText = "a URI [RFC3986](https://tools.ietf.org/html/rfc3986) or\n"
```

Comment:
Needs to be put in a separate test because the current is related with backticks.

Reply:
> the current is related with backticks

What's you mean?

Reply:
It's for content containing Markdown backticks \`\`.